hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesState.java

```diff
@@ -323,16 +325,6 @@ public AddressBook updateWeight(@NonNull AddressBook configAddressBook, @NonNull
         return configAddressBook;
     }
 
-    private void resetDefaultMerkleDbPathIfNeeded() {
-        // This is a workaround to support multiple nodes running in a single process for testing
```

Comment:
I think this comment is worth saving. Maybe move it to where resetDefaultInstancePath() is called in the constructor.

Reply:
I moved this comment to javadoc of `MerkleDb#resetDefaultInstancePath`, so the information is not lost.